tools/dockerise/__init__.py

```diff
@@ -170,12 +81,18 @@ def platforms():
 
 
 def get_selected_platform():
+    # Get information about the selected image
     try:
-        selected = client.images.get("{}:selected".format(repository))
+        img_info = json.loads(
+            subprocess.Popen(
+                ["docker", "image", "inspect", "{}:selected".format(repository)], stdout=subprocess.PIPE
+            ).communicate()[0],
```

Comment:
No need for all of this here, you can just use subprocess.check_output
```suggestion
            subprocess.check_output(["docker", "image", "inspect", "{}:selected".format(repository)]),
```

Reply:
It doesn't suppress the output if we just use check_output?

Reply:
I thought it would capture output, is there some stderr output that's not getting captured?

Reply:
Also, check_output wont let me set the stdout argument

Reply:
I decided not to capture stderr so we can see the errors if there are any, but we dont want the json output of docker inspect dumping into the terminal every time you run a b command